File: 1291/solution.java
class Solution {
    public List<Integer> sequentialDigits(int low, int high) {
        int minDigit = (int)Math.floor(Math.log10(low))+1;
        int maxDigit = (int)Math.floor(Math.log10(high))+1;
        String s = "123456789";
        List<Integer> res = new ArrayList<>();
        for(int l=minDigit;l<=maxDigit;l++){
            for(int i=0;i<10-l;i++){
                int value = Integer.parseInt(s.substring(i,i+l));
                if(value>high)
                    break;
                if(value>=low)
                    res.add(value);
            }
        }
        return res;
    }
}

File: 1291/solution.py
class Solution:
    def sequentialDigits(self, low: int, high: int) -> List[int]:
        l = len(str(low))
        u = len(str(high))
        s = '123456789'
        res = []
        for i in range(l, u+1):
            for j in range(10-i):
                k = int(s[j:j+i])
                if k>high:
                    break
                if k>=low:
                    res.append(k)
        return res
